azure-spring-boot/src/test/java/com/microsoft/azure/keyvault/spring/KeyVaultOperationUnitTest.java

```diff
@@ -90,4 +94,34 @@ public String nextPageLink() {
             return Arrays.asList(mockSecretItem);
         }
     }
+
+    @Test
+    public void should_never_fetch_listSecrets_when_policy_is_single() {
+        final KeyVaultOperation keyVaultOperation = new KeyVaultOperation(keyVaultClient,
+                fakeVaultUri,
+                Constants.DEFAULT_REFRESH_INTERVAL_MS,
+                VaultPolicy.SINGLE);
+
+        final Object result = keyVaultOperation.get(testPropertyName1);
+
+        assertThat(result).isEqualTo(testPropertyName1);
+        Mockito.verify(keyVaultClient, never()).listSecrets(anyString());
+    }
+
+    @Test
+    public void should_call_fetch_listSecrets_when_policy_is_single() {
```

Comment:
Test case never typo?  `should_call_fetch_listSecrets_when_policy_is_single` -> `should_call_fetch_listSecrets_when_policy_is_list`?

Reply:
oh.. is my fault.